parser/fsimporter.go

```diff
@@ -568,16 +569,17 @@ func (fs *FSImporter) parseFiles(indexedFiles []*fpt.IndexedFile, parsingThreads
 								}
 							}
 
-							// increment txt query count for host in uconn
-							if queryTypeName == "TXT" {
-								// get destination for dns record
-								dst := parseDNS.Destination
-								dstIP := net.ParseIP(dst)
-
-								// Run conn pair through filter to filter out certain connections
-								ignore := fs.filterConnPair(srcIP, dstIP)
-								if !ignore {
+							// get destination for dns record
+							dst := parseDNS.Destination
+							dstIP := net.ParseIP(dst)
 
+							// Run conn pair through filter to filter out certain connections
+							ignore := fs.filterConnPair(srcIP, dstIP)
```

Comment:
Do we want to allow DNS for internal-> internal? It looks like we are keeping the filter for dns logs currently.

Reply:
The change for removing the filter is in #608.

Reply:
:+1: Pulled in #608.